lib/logstash/codecs/protobuf.rb

```diff
@@ -0,0 +1,189 @@
+# encoding: utf-8
+require 'logstash/codecs/base'
+require 'logstash/util/charset'
+require 'protocol_buffers' # https://github.com/codekitchen/ruby-protocol-buffers
+
+class LogStash::Codecs::Protobuf < LogStash::Codecs::Base
```

Comment:
Can you add more documentation on top of this line on how to use this codec? This gets converted into the rendered docs and really useful for users.